### PR TITLE
Set AWS PCA to use short-lived mode

### PR DESCRIPTION
**Pull request by MaggieXJZhang**
There is a cheaper option $50 vs $400 for enabling certificates in AWS. We should offer and default to the cheaper option.

background: 
* https://aws.amazon.com/private-ca/pricing/
* https://docs.aws.amazon.com/privateca/latest/userguide/short-lived-certificates.html

### CODE REVIEW

File: eks/examples/cnpack/README.md [hunk 1]
```diff
@@ -224,8 +224,9 @@ No requirements.
 | <a name="input_cluster_name"></a> [cluster\_name](#input\_cluster\_name) | Name of the cluster | `string` | n/a | yes |
 | <a name="input_common_name"></a> [common\_name](#input\_common\_name) | Common Name for PCA Creation | `string` | `"cluster.local"` | no |
 | <a name="input_fluentbit_enabled"></a> [fluentbit\_enabled](#input\_fluentbit\_enabled) | Set to true to enable, false to disable | `bool` | `true` | no |
-| <a name="input_metrics_server_enabled"></a> [metrics\_server\_enabled](#input\_metrics\_server\_enabled) | Set to true to enable the network support for Metrics Server, false to disable | `bool` | `false` | no |
+| <a name="input_metrics_server_enabled"></a> [metrics\_server\_enabled](#input\_metrics\_server\_enabled) | Set to true to enable the network support for Metrics Server, false to disable | `bool` | `true` | no |
```
Comment (ikatz-nvidia) [inline eks/examples/cnpack/README.md:227]: Is this a typo fix? I don't see a corresponding change in the code

File: eks/examples/cnpack/aws-pca.tf [hunk 2]
```diff
@@ -9,6 +9,7 @@ AWS Private Cert Authority Config
 resource "aws_acmpca_certificate_authority" "cnpack-pca" {
   count = var.pca_enabled ? 1 : 0
   type  = "ROOT"
+  usage_mode = var.pca_short_lived ? "SHORT_LIVED_CERTIFICATE" : "GENERAL_PURPOSE"
```
Comment (ikatz-nvidia) [inline eks/examples/cnpack/aws-pca.tf:12]: Seems to agree with https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/acmpca_certificate_authority#usage_mode

File: eks/examples/cnpack/README.md (hunk 1 above)
Comment (MaggieXJZhang) [reply eks/examples/cnpack/README.md:227]: yes a typo fix, the docs are auto generated from `terraform-docs markdown .` and this is probably missed in some previous commit.

File: eks/examples/cnpack/aws-pca.tf (hunk 2 above)
Comment (MaggieXJZhang) [reply eks/examples/cnpack/aws-pca.tf:12]: yup that's what I referred to when I wrote this change

**Comment by slbryson** [comment]
Looks Good

**Review by ikatz-nvidia — APPROVED** [review]

**Review by slbryson — APPROVED** [review]

**Review by evberrypi — APPROVED** [review]
LGTM thank you!

**Pull request merged**